docker/docker-compose-3/api.py

```diff
@@ -0,0 +1,74 @@
+import mysql.connector
+from mysql.connector import Error
+from flask import Flask
+import os
+from dotenv import load_dotenv
+from flask import request, jsonify
+
+app = Flask(__name__)
+
+load_dotenv()
+
+def connect():
+    try:
+        connection = mysql.connector.connect(host=os.getenv("MYSQL_HOST"),
+                                            database=os.getenv("MYSQL_DB"),
+                                            user=os.getenv("MYSQL_USER"),
+                                            password=os.getenv("MYSQL_PASSWORD"),
+                                            port=3306)
+        if connection.is_connected():
+            return connection
+
+    except Error as error:
+        print("Error while connecting to MySQL", error)
+
+def add_db():
```

Comment:
if you have only one table, you can add in once, no need to try to create each request